out/release/release_finder_test.go

```diff
@@ -0,0 +1,56 @@
+package release_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/pivotal-cf/go-pivnet/v7"
+	"github.com/pivotal-cf/pivnet-resource/v3/out/release"
+	"github.com/pivotal-cf/pivnet-resource/v3/out/release/releasefakes"
+
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ReleaseFinder", func() {
+	var (
+		pivnetClient *releasefakes.FinderClient
+		finder       release.ReleaseFinder
+		releaseID    int
+		pivnetRelease pivnet.Release
+		finderErr error
+		productSlug string
+	)
+	BeforeEach(func() {
+		pivnetClient = &releasefakes.FinderClient{}
+		finderErr = nil
+		productSlug = "special-product"
+	})
+
+	Describe("Find", func() {
+
+		Context("", func() {
```

Comment:
is this a placeholder for additional tests?

Reply:
It was a reminder for potential errors, but isn't needed. Removing.